LICENSE.txt

```diff
@@ -0,0 +1 @@
+Copyright (c) 2018 Strong Analytics, LLC. All Rights Reserved.
```

Comment:
Should this be updated (e.g., 2022)? I have no idea if that matters.

Reply:
I'm not sure either, but since it's new I will update it to 2022. It looks like strong-opx has 2021 so I guess it gets updated over time.